packages/webapp/webapp_server.js

```diff
@@ -439,9 +440,9 @@ var getUrlPrefixForArch = function (arch) {
     '' : '/' + '__' + arch.replace(/^web\./, '');
 };
 
-// parse port to see if its a Windows Server style named pipe. If so, return as-is (String), otherwise return as Int
+// parse port to see if it's a Windows Server-style named pipe or a UNIX path/filename for a UNIX domain socket file. If so, return as-is (String), otherwise return as Int
 WebAppInternals.parsePort = function (port) {
-  if( /\\\\?.+\\pipe\\?.+/.test(port) ) {
+  if( /\\\\?.+\\pipe\\?.+/.test(port) || /^(.+)\/([^/]+)$/.test(port) ) {
```

Comment:
I'm not convinced this new RegExp is much better than just checking to see if the path has a `/` in it at all (i.e. `/\//`, which is still not ideal.  As it stands, this wouldn't allow the socket file to be in the current directory, something we should probably try to support via [`path.resolve`](https://nodejs.org/dist/latest-v4.x/docs/api/path.html#path_path_resolve_from_to), for example.

In addition to that, can we also could we consider another option for detecting if it's a TCP socket file?  This should also probably check `process.platform` (to ensure it's `win32`) for the first part.